curriculum/challenges/english/25-front-end-development/quiz-javascript-variables-and-data-types/66edc25ae5ea80bf6f785552.md

```diff
@@ -220,7 +215,7 @@ No. It's recommended not to use semicolons in JavaScript.
 
 #### --answer--
 
-No, but it is generally recommended to use semicolons consistently to avoid potential issues in code.
+No, semicolons are added automatically.
```

Comment:
I think this should replace `Semicolons are sometimes required in JavaScript.`, not the correct answer

Reply:
Any suggestions?

What about, "No, semi colons will cause errors." or "Only when followed by a new line."?

Reply:
My initial suggestion was just to replace the "Semicolons are sometimes required in JavaScript" distractor with something else. It could be anything, it doesn't really matter as long as it is wrong with no ambiguity.

The "No, semicolons will cause errors." would work.

---

But I still maintain the question

`Are semicolons required in JavaScript?`

Is the problem, because we want "no" as the correct answer, but really it is in fact "yes sometimes" they are required.